dojo/settings/unittest.py

```diff
@@ -2,13 +2,13 @@
 # first;
 # Do so by copying the file dojo/settings/settings.dist.py to
 # dojo/settings/settings.py; Otherwise, the following import will not work
-from .settings import *
+# from .settings import *
```

Comment:
I know this is likely redundant, but is probably worth keeping

Reply:
Uncomment line in this commit https://github.com/DefectDojo/django-DefectDojo/pull/8320/commits/586e450a15dbd850adccd50b095b306374bcdecf

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8320/commits/cf26b9343174595697195bd93067a56a13fc9bda